packages/contentstack-migration/src/utils/error-handler.js

```diff
@@ -1,21 +1,19 @@
 'use strict';
 
-const { error } = require('./logger');
 const { errorMessageHandler } = require('./constants');
 
 module.exports = (data, type, method, err) => {
-  if (data && type && method) {
-    error(`Error occurred while ${errorMessageHandler[method]} ${type}: ${data}.`);
-  }
-
-  if (err.errorMessage) {
-    error(err.errorMessage);
-  }
-  if (err instanceof Error && err && err.message && err.stack) {
-    error(err.message);
-    // error(err.stack)
-  } else {
-    error(err);
-  }
+  // NOTE: Commenting this code as the API errors are handled through error-helper.js also
+  // if (data && type && method) {
+  //   logger.log('error', { error: `Error occurred while ${errorMessageHandler[method]} ${type}: ${data}.` });
+  // }
+  // if (err.errorMessage) {
+  //   logger.log('error', { errorAPI: err.errorMessage });
+  // }
+  // if (err instanceof Error && err && err.message && err.stack) {
+  //   logger.log('error', { error: err.message });
+  // } else {
+  //   logger.log('error', { error: err });
+  // }
```

Comment:
Is this code no longer required? If so we can remove the code

Reply:
Deleted the file